webpack.config.js

```diff
@@ -101,7 +102,9 @@ module.exports = [{
   },
   postcss: function() {
     return [
-      require('autoprefixer')
+      require('autoprefixer'),
+      require('postcss-custom-properties'),
```

Comment:
will postcss-custom-properties actually remove the custom properties from the output and substitute them with hard-coded values? If so, I'm not sure we actually want to use this transformation, since we don't want custom variables in the output code.